dallinger/experiment_server/experiment_server.py

```diff
@@ -319,7 +319,10 @@ def advertisement():
         status = part.status
     except exc.SQLAlchemyError:
         status = None
-
+    try:
+        app_id = config.get('id')
```

Comment:
Related to this, there are two scenarios where you'd call `config.get()` with a default:
1. You are going use the default value you pass if nothing is stored in the `config` (this is the case @davisagli is concerned about)
2. You need to check if a config value is defined to determine whether to do something, but it's perfectly expected behavior that the value might not be set

Requiring a try/catch for second case feels onerous to me.